.env.example

```diff
@@ -32,3 +32,12 @@ HSES_DATA_PASSWORD=password
 # In production, Sequelize instance is created with a postgres URI.
 # This URI is automatically dropped into the cloud.gov environment as the env variable DATABASE_URL
 DATABASE_URL=secret
+# Local development variables to support s3 endpoint development and testing
+LOCAL_DEV=true
+# Uncomment the following if you aren't using docker for development. Note: This requires minio to be running locally on port 9000
+# S3_ENDPOINT=http://localhost:9000
+S3_BUCKET=ttadp-test
+AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+MINIO_ROOT_PASSWORD=$AWS_SECRET_ACCESS_KEY
```

Comment:
These weren't evaluated for me, I had to login with the literal value "$AWS_SECRET_ACCESS_KEY"

Reply:
Interesting. I have changed the .env.example file to set them independently, I wonder why it wasn't evaluated.